ogr/services/pagure/pull_request.py

```diff
@@ -105,6 +107,14 @@ def created(self) -> datetime.datetime:
     def diff_url(self) -> str:
         return f"{self.url}#request_diff"
 
+    @property
+    def patch(self) -> bytes:
+        patch_url = f"{self.url}.patch"
+        response = get(patch_url)
```

Comment:
I am only if we can use the session used across the whole codebase. (`self.__call_api`)

Reply:
`self.__call_api` automatically uses API endpoint, also `patch_url` points to raw file not JSON
maybe  `_call_project_api_raw`?

Reply:
Updated, can you check again?